wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -124,6 +130,39 @@ def _check_modulo_patterns(
         if self._modulo_string_pattern.search(text_data):
             self.add_violation(consistency.ModuloStringFormatViolation(node))
 
+    def _check_complex_f_string(self, node: ast.JoinedStr) -> None:
+        # Whitelists all simple uses of f strings
+        # Checks if list, dict, function call with no parameters or variable
+        for string_component in node.values:
+            if isinstance(string_component, ast.FormattedValue):
+                if self._valid_f_value(string_component):
```

Comment:
We tend to use `_is_valid_...` naming style
I would also suggest to name it `_is_valid_formatted_value`